<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmClock</class>
 <widget class="QWidget" name="AlarmClock">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 25pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>闹钟</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="leftLcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="rightLcdNumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="lAddButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lSubButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rAddButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rSubButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>21</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>20</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 48pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setAlarmButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>172</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置闹钟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleAlarmButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>172</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消闹钟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectBellButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>212</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>铃声选取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="repetitionNumberButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重复次数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
